stable/redis/templates/deployment.yaml

```diff
@@ -21,6 +21,14 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       {{- end }}
+      initContainers:
+      - name: "fix-non-root-permissions"
+        image: "busybox"
+        imagePullPolicy: "Always"
+        command: [ "chmod", "-R", "g+rwX", "/bitnami" ]
+        volumeMounts:
+        - name: redis-data
+          mountPath: /bitnami
```

Comment:
The `bitnami/redis` image now uses a non-root approach where the docker image has the `USER` instruction. The init container here make the volume mount `/bitnami` writable by the default container user.

Reply:
@sameersbn weren't we going to change this to fsGroup?

Reply:
@prydonius Ideally we should be defining `securityContext.runAsUser=1001` and `securityContext.fsGroup=1001` but we have seen issues deploying the chart on minikube which should be fixed in the next release as https://github.com/kubernetes/minikube/pull/1991 fixes the issue. Until the next minikube release I decided to use initContainers to allow writing to the volume.

Reply:
just noticed that a new version of minikube has already been released. I'll test out the changes on minikube and update the PR

Reply:
minikube 0.22.3 fixes the deployment issue. Have updated the PR to define the `securityContext` and removed the use of `initContainers`